ui/app/src/components/SiteSwitcherSelect/SiteSwitcherSelect.tsx

```diff
@@ -65,18 +66,13 @@ function SiteSwitcherSelect(props: SiteSwitcherSelectProps) {
 	return (
 		<Select
 			displayEmpty
-			variant="standard"
+			variant="outlined"
```

Comment:
Updated the select component variant to 'outlined', since the 'standard' variant doesn't have focused styles. To keep the styling of the component, I removed the border when not focused